### PR TITLE
Test PR for actions fix

### PR DESCRIPTION
**Pull request by minsulee2**
The problem was the base image ubuntu-latest has updated from 18.04 to 20.04. All integration tests have passed in ubuntu-18.04 or macos-latest without any errors. 

Updates:
- Use macos-latest image for test env
- Use yarn
- Add some async await just in case

### CODE REVIEW

**Comment by minsulee2** [comment]
PTAL

**Review by platfowner — APPROVED** [review]
LGTM
Thanks for handling this issue!

File: .github/workflows/github-actions.yml [hunk 1]
```diff
@@ -14,27 +14,25 @@ on:
...
+    runs-on: macos-latest
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
       # Setup node environment for testing
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
         with:
-          node-version: '12.16'
+          node-version: '12.x'
```
Comment (liayoo) [inline .github/workflows/github-actions.yml:24]: question) is it ok to not fix it to v12.16?

Comment (minsulee2) [reply .github/workflows/github-actions.yml:24]: Nah, It is ok with fix #419 Thanks for asking!

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Comment by minsulee2** [comment]
Thanks for the reviews! merging it now

**Pull request merged**